02-peer-protocol.md

```diff
@@ -198,6 +211,9 @@ acceptance of the new channel.
 #### Requirements
 
 
+The receiving MAY reject the channel if the `chain-hash` value within the
```

Comment:
MUST.  You can't really do anything here, and you certainly can't ignore the field (otherwise you'll get weird failures later).